Orange-OpenSource/its-client
# Rust configuration

New features
-----------------

* New `client` module
* Information struct is now a Message enumeration variant
* Information no longer manages the region of responsibility
* Configuration has been split into `Configuration` and `NodeConfiguration`
* NodeConfiguration is an optional part of `Configuration` which holds server instance information
* Fixed warnings in documentation (closes #106)

**Note: `exchange.rs` contains commented code that will be either deleted or updated in further commits/PR**

How to test
---------------

Nothing much to do other than checking that build, tests and benches runs correctly
In addition from previous PR also build the documentation to check it correctly does with no warnings

1. `cargo clean`
2. `cargo build`
3. `cargo test`
4. `cargo bench`
5. `cargo doc --no-deps --open`
